Sources/Utils/Backtrace.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2014-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the University of Illinois/NCSA Open
+// Source License found in the LICENSE file in the root directory of this
+// source tree. An additional grant of patent rights can be found in the
+// PATENTS file in the same directory.
+//
+
+#include "DebugServer2/Utils/Log.h"
+
+#include <string>
+#if defined(OS_DARWIN) || defined(__GLIBC__)
+#include <cxxabi.h>
+#include <execinfo.h>
+#endif
+
+namespace ds2 {
+namespace Utils {
+
+#if defined(OS_DARWIN) || defined(__GLIBC__)
+void PrintBacktrace() {
+  static const int kStackSize = 100;
+  static void *stack[kStackSize];
+  int stackEntries = backtrace(stack, kStackSize);
+  char **stackSymbols = backtrace_symbols(stack, stackEntries);
```

Comment:
Not bad. I think I'll use `dladdr` yeah. Seems cleaner and provides more information.